futures-util/src/future/flatten_sink.rs

```diff
@@ -0,0 +1,81 @@
+use core::fmt;
+
+use futures_core::{task::Context, Async, Future};
```

Comment:
This syntax (the nested multi-element path inside a group) wasn't around in in 1.20, and we try to stay backcompat for the last several major versions of Rust. Can you update to either have a separate import for `futures_core::task::Context` or import `futures::core{task, Async, Future}` and use `task::Context`?

Reply:
Fixed.